tests/components/aladdin_connect/test_sensor.py

```diff
@@ -83,3 +96,71 @@ async def test_sensors(
 
         state = hass.states.get("sensor.home_wi_fi_rssi")
         assert state
+
+
+async def test_sensors_model_02(
+    hass: HomeAssistant,
+    mock_aladdinconnect_api: MagicMock,
+) -> None:
+    """Test Sensors for AladdinConnect."""
+    config_entry = MockConfigEntry(
+        domain=DOMAIN,
+        data=CONFIG,
+        unique_id="test-id",
+    )
+    config_entry.add_to_hass(hass)
+
+    await hass.async_block_till_done()
+
+    with patch(
+        "homeassistant.components.aladdin_connect.AladdinConnectClient",
+        return_value=mock_aladdinconnect_api,
+    ):
+        mock_aladdinconnect_api.get_doors = AsyncMock(
+            return_value=[DEVICE_CONFIG_MODEL_02]
+        )
+        await hass.config_entries.async_setup(config_entry.entry_id)
+        await hass.async_block_till_done()
+
+        registry = entity_registry.async_get(hass)
+        entry = registry.async_get("sensor.home_battery_level")
+        assert entry
+        assert entry.disabled is False
+        assert entry.disabled_by is None
+        state = hass.states.get("sensor.home_battery_level")
+        assert state
+
+        entry = registry.async_get("sensor.home_wi_fi_rssi")
+        await hass.async_block_till_done()
```

Comment:
`registry.async_get("sensor.home_battery_level")` doesn't have a block till done, so I'm assuming this doesn't need one either?